examples/graph_animations/example2.md

```diff
@@ -0,0 +1,196 @@
+## Depth First Search
+
+### Points covered
+1. Graph creation using `LightGraphs.jl`
+2. Demonstrate additional utility functions
+
+### Graph creation
+
+In the previous example, it was shown how a graph can be created and animated for a simple graph data type. To simplify a lot of details one can provide a known graph type i.e. from JuliaGraphs. The candidate types for this are:
+* `SimpleGraph`
+* `SimpleDiGraph`
+* `SimpleWeightedGraph`
+* `SimpleWeightedDiGraph`
+
+Here I cover the case when it is represented using `SimpleGraph` from the LightGraphs package.
+
+```julia
+using LightGraphs
+g = SimpleGraph(6)
+add_edge!(g, 1, 2)
+add_edge!(g, 1, 3)
+add_edge!(g, 2, 4)
+add_edge!(g, 3, 5)
+add_edge!(g, 4, 6)
+
+ag, nodes, edges = create_graph(g, 300, 300; layout=:spring, mode=:static)
+```
+`ag` - A Javis object storing some useful meta-data corresponding to the graph. Only the translate operation is supported on it as of now.
+
+`nodes` - list of references to node objects in the order of node id
+
+`edges` - list of references to edge objects in the order of creation
+
+The `layout` defines how the nodes shall be arranged on the canvas. The `mode` argument determines whether the graph is to be considered a static or a dynamic graph.
+
+#### Static graphs
+* The layout computation is done only once when the entire graph is known.
+* Updates to nodes/edge properties in the input graph are not animated i.e. the final graph state is used for the animation
+
+#### Dynamic graphs
+* Addition of each new node leads to recomputation of new layout
+* Group animations of nodes/edges are separated across frames using a predefined ordering
+* Updates to edges and nodes through properties are animated with the help of action transitions leading to visualisation for time evolving graphs
+
+*Note - Dynamic layout will be much more computationally intensive than static layout*
+
+**Adding or removing nodes and edges**
+
+New nodes and edges can be added to the canvas or existing nodes deleted after the graph creation.
+```julia
+removeNode!(ag, 2)
+addNode!(ag, 7)
+addEdge!(ag, 1, 7)
```

Comment:
we might want to think about other names here to make clear that this has something to do with Javis. Maybe even just `Javis.add_edge!`